docs/projects/lottie/supported.rst

```diff
@@ -362,6 +362,50 @@ Switch Group Layer
 | Active Layer Name |        |tick|       |       |tick|       |
 +-------------------+---------------------+--------------------+
 
+Linear Gradient Layer
+---------------------
++---------------------+---------------------+---------------------------------------------+
+|       Property      |        Value        |               Type(Animation)               |
++=====================+=====================+=============================================+
+|       Z depth       |        |tick|       |                  |cross|                    |
++---------------------+---------------------+---------------------------------------------+
+|       Opacity       |        |tick|       |                  |cross|                    |
++---------------------+---------------------+---------------------------------------------+
+|     Blend_method    |        |cross|      |                  |cross|                    |
```

Comment:
Please check other supported features and use "Partially supported" and a "dash" for consistency.